pyproject.toml

```diff
@@ -75,5 +75,7 @@ module = [
     "pytorch_lightning.utilities.parsing",
     "pytorch_lightning.utilities.seed",
     "pytorch_lightning.utilities.xla_device",
+    "pytorch_lightning.compressor.intel.neural_compressor.quantization",
+    "pytorch_lightning.compressor.intel.neural_compressor.utils",
```

Comment:
Let's move this under `pytorch_lightning/utilities/quantization.py` as a single file.

Reply:
Done